components/chart/types/rangebar.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Range Bar
+page_title: Chart - Range Bar
+description: Overview of the Range Bar Chart for Blazor.
```

Comment:
Try to write descriptions that are around 150 symbols and describe the content of the article.